projects/demo/src/modules/components/swipe-action/examples/1/index.html

```diff
@@ -20,22 +20,28 @@
         tuiIconButton
         tuiSwipeAction
         type="button"
-    ></button>
+    >
+        Eye
```

Comment:
Let's say Show, Edit and Delete, instead of icon names here for all swipe-actions examples, so it looks like real case

Reply:
fixed